tests/rest_api_tests.py

```diff
@@ -1,7 +1,9 @@
 """Unit Tests for Rest API."""
 
+import sys
 import unittest
-import rest_api
+sys.path.append('../')   # pylint diable=g-import-not-at-top
```

Comment:
still don t understand why this is needed.

Reply:
Technically, it isn't. The only reason it is there is to allow the tests to be run from the tests/ folder, rather than having to move them out of the folder to the same directory as the file which is to be tested/imported